frontend/styles/components/organisms/ViewMeeting.module.scss

```
@@ -1,10 +1,25 @@
+@import '../../Variables.module.scss';
```

Stylelint (17.14.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)